### PR TITLE
conversion to jupyer/colab

### PR DESCRIPTION
**Pull request by merla18**
plus bug fixes

### CODE REVIEW

File: tables/automl/notebooks/purchase_prediction/purchase_prediction.ipynb [hunk 1]
```diff
@@ -1,909 +1,1242 @@
...
+   "metadata": {},
+   "source": [
+    "### Import libraries and define constants\n",
+    "\n",
+    "First, import Python libraries required for training,\n",
+    "The code example below demonstrates importing the AutoML Python API module into a python script. "
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (gguuss) [inline tables/automl/notebooks/purchase_prediction/purchase_prediction.ipynb]: this alignment looks odd, I'm unfamiliar with conventions advocating for 3 spaces.

File: tables/automl/notebooks/purchase_prediction/purchase_prediction.ipynb [hunk 2]
```diff
@@ -1,909 +1,1242 @@
...
+    "colab_type": "code",
+    "id": "Iu3KNlcwsRhN"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Create dataset { vertical-output: true, output-height: 200 }\n",
+    "\n",
+    "dataset_display_name = 'colab_trial1' #@param {type: 'string'}\n",
+    "\n",
+    "create_dataset_response = client.create_dataset(\n",
```
Comment (gguuss) [inline tables/automl/notebooks/purchase_prediction/purchase_prediction.ipynb]: Is there a test for this?

File: tables/automl/notebooks/purchase_prediction/purchase_prediction.ipynb [hunk 3]
```diff
@@ -1,909 +1,1242 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "RQuGIbyGgud9"
+   },
+   "outputs": [],
+   "source": [
+    "#@title Create bucket to store data in { vertical-output: true, output-height: 200 }\n",
```
Comment (gguuss) [inline tables/automl/notebooks/purchase_prediction/purchase_prediction.ipynb]: How do we know this code works without manually testing it?

**Comment by sirtorry** [comment]
I agree with @gguuss, we'll eventually want to set up CI/CD to test our notebooks. However, since this notebook is strictly an improvement over the old version, let's merge it.

**Pull request merged**